front/README.md

```diff
@@ -250,4 +250,13 @@ All common code (and shared components) supposed to be in `common/`.
 - Better docs: still in use ? with
 - jsdocs
 
-### Module editor
+### Updating Dependencies
+When `package.json` changes (new packages or updated versions), developers may have issues running the app with Docker. New packages or versions might not be recognized by Docker.
+
+To fix this, follow these steps:
+
+1. After pulling new changes, run `yarn install` to update local dependencies.
+2. If issues persist, delete `node_modules` and run `yarn install` again.
+3. Run `docker compose down osrd-front && docker system prune --volumes && docker compose up -d --build osrd-front` to rebuild Docker images with new dependencies.
```

Comment:
As discussed, the first part was only there to ensure `front` container was down before `prune` (which is a too big hammer for this, but was OK for me).
So I'd go for a single more narrow command on this one:
```suggestion
3. Run `docker compose build --no-cache front` to rebuild Docker images from scratch with new dependencies.
```